### PR TITLE
Topic/remove bus id unknown

### PR DESCRIPTION
**Pull request by Akshay-Venkatesh**

### CODE REVIEW

**Comment by brminich** [comment]
ok to test

**Comment by Akshay-Venkatesh** [comment]
bot:pipe:retest

**Comment by brminich** [comment]
issue is #5145 
bot:pipe:retest

File: test/gtest/ucs/test_topo.cc [hunk 1]
```diff
@@ -14,42 +14,31 @@ class test_topo : public ucs::test {
 
 UCS_TEST_F(test_topo, find_device_by_bus_id) {
     ucs_status_t status;
-    ucs_sys_device_t unknown_dev1;
-    ucs_sys_device_t unknown_dev2;
-    ucs_sys_bus_id_t dummy_bus_id1;
+    ucs_sys_device_t current_index;
+    ucs_sys_device_t unknown_dev;
```
Comment (yosefe) [inline test/gtest/ucs/test_topo.cc:18]: the names current_index and unknown_dev seem wrong
maybe it should be just dev1, dev2?

File: test/gtest/ucs/test_topo.cc [hunk 2]
```diff
@@ -14,42 +14,31 @@ class test_topo : public ucs::test {
...
-    ASSERT_EQ(unknown_dev2, 1);
+    ASSERT_EQ(unknown_dev, (current_index + 1));
 }
 
 UCS_TEST_F(test_topo, get_distance) {
     ucs_status_t status;
-    ucs_sys_device_t unknown_dev1;
-    ucs_sys_device_t unknown_dev2;
+    ucs_sys_device_t unknown_dev1 = UCS_SYS_DEVICE_ID_UNKNOWN;
+    ucs_sys_device_t unknown_dev2 = UCS_SYS_DEVICE_ID_UNKNOWN;
```
Comment (yosefe) [inline test/gtest/ucs/test_topo.cc:39]: maybe just do
```
 status = ucs_topo_get_distance(UCS_SYS_DEVICE_ID_UNKNOWN, 
                                UCS_SYS_DEVICE_ID_UNKNOWN, 
                                &distance);
 EXPECT_EQ(UCS_OK, status)
```

**Comment by brminich** [comment]
bot:pipe:retest

**Pull request merged**